gui/notesView.py

```diff
@@ -0,0 +1,41 @@
+import wx
+
+import service
+import gui.globalEvents as GE
+import gui.display as d
+import gui.mainFrame
+
+class NotesView(d.Display):
+    DEFAULT_COLS = []
+
+    def __init__(self, parent):
+        self.lastFitId = None
+        d.Display.__init__(self, parent, style=wx.BORDER_NONE)
+        self.mainFrame = gui.mainFrame.MainFrame.getInstance()
+        mainSizer = wx.BoxSizer(wx.VERTICAL)
+        self.editNotes = wx.TextCtrl(self, style=wx.TE_MULTILINE)
+        mainSizer.Add(self.editNotes, 1, wx.EXPAND)
+        self.SetSizer(mainSizer)
+        self.mainFrame.Bind(GE.FIT_CHANGED, self.fitChanged)
+        self.Bind(wx.EVT_TEXT, self.onText)
+
+    def fitChanged(self, event):
+        sFit = service.Fit.getInstance()
+        fit = sFit.getFit(event.fitID)
+
+        self.Parent.Parent.DisablePage(self, not fit or fit.isStructure)
+
+        if event.fitID is None and self.lastFitId is not None:
```

Comment:
Fair enough.  Probably something that could be optimized, but not worth worrying about right now.